server/controllers/buyer/getFavoriteHandler.js

```diff
@@ -0,0 +1,10 @@
+const { getFavorite } = require('../../database/queries');
+
+const getFavoriteHandler = (req, res, next) => {
+  const { userId } = req.params;
+  getFavorite(userId)
+    .then(({ rows }) => res.json({ status: 200, data: rows }))
+    .catch(next);
+};
+
+module.exports = getFavoriteHandler;
```

Comment:
please handle these two cases in your route:
1- when the favorite for the given user id is empty
```
res.json({res.json({
status:404,
message: 'favorite list is empty',
data: [ ]
})})
```

2- when the given user id is not exists
```
res.json({res.json({
status:404,
message: 'user doesn't exist',
data: [ ]
})})
```